bin/install.sh

```diff
@@ -111,12 +111,14 @@ fishFunctionsDir="${PREFIX}/fish_functions"
 mkdir -p "${fishFunctionsDir}"
 cat <<'EOF' >"${fishFunctionsDir}/okta-aws.fish"
 function okta-aws
-    withokta "aws --profile $argv[1]" $argv
+    set -lx OKTA_PROFILE "$argv[1]"
+    OKTA_PROFILE="$1" withokta "aws --profile $argv[1]" $argv[2..-1]
 end
 EOF
 cat <<'EOF' >"${fishFunctionsDir}/okta-sls.fish"
 function okta-sls
-    withokta "sls --stage $argv[1]" $argv
+    set -lx OKTA_PROFILE "$argv[1]"
+    OKTA_PROFILE="$1" withokta "sls --stage $argv[1]" $argv[2..-1]
```

Comment:
I'm not sure what this line will do.  From what I learned of fish, line 120 is the one to keep, and remove `OKTA_PROFILE="$1"` from this line.

Reply:
fixed